docs/tutorials/js-sdk-tutorial.md

```diff
@@ -0,0 +1,144 @@
+---
+Title: Datahub JS SDK Tutorial
+---
+
+## Introduction
+
+The [DataHub](http://datahub.io/) platform stores a lot of different datasets - which are packages of useful data alongside with the description (here is [the dataset specification](https://frictionlessdata.io/docs/data-package/)). The data, stored on the DataHub, has a nice structure, views and a description that help people to get insights.
+You can also store and share your own datasets on the DataHub
```

Comment:
You'll need white space here if this is a new paragraph. Otherwise, there's no actual use - it will still remain on the same line

Reply:
fixed